shudder/__main__.py

```diff
@@ -38,10 +41,11 @@ def receive_signal(signum, stack):
 def run_commands():
     for command in CONFIG["commands"]:
         try:
+            statsd.gauge(HEARTBEAT_METRIC, 1)
```

Comment:
mm - reason for sending metric here and there? 

you're using var that was initialized in __main__ - bit problematic

you might do to either:
1. initialize statsd in the upper part (before the code)
2. remove the gauge from here..


+ What about tags? we get all of them from the host so we're good? :)